api/api.go

```diff
@@ -270,6 +270,10 @@ func (c *Client) Status(trackID string) (*StatusInfo, error) {
 
 	defer res.Body.Close()
 
+	if _, err := c.List(trackID); err != nil {
+		return nil, err
+	}
+
```

Comment:
/tracks/:track_id/status is the endpoint for a problem named "status" which we don't have.